fusion-studio-extension/src/browser/widget/_welcome.scss

```diff
@@ -9,4 +9,80 @@
       text-decoration: underline;
     }
   }
-}
\ No newline at end of file
+}
+
+// fusion-logo style
+.fusion {
+  &-header {
+    align-items: center;
+    display: flex;
+  }
+  &-logo {
+    // i feel bad for this monstrosity but without it, it doesnt work 
```

Comment:
@marmoure When leaving comments can you prefix them with either `NOTE(initials)` or `TODO(initials)` as it helps in future for us to know who made the change if we need to ask questions about it.

e.g.
```
// NOTE(YB) I feel bad for this monstrosity but without it, it doesn't work

```

@ccheraa Any thoughts on this?